src/missions/use-after-free-control-flow/README.md

```diff
@@ -1,3 +1,123 @@
 # Exploiting heap use-after-free to manipulate control flow
 
-*This section remains under implementation.*
+**This mission applies only on CHERI-RISC-V builds with temporal safety!** The
+executable is *quite* exploitable on RISC-V without CHERI and is *still
+exploitable* (though in fewer ways) on CHERI-RISC-V without a temporally safe
+heap allocator.
```

Comment:
Do we want to be so explicit, or do we want to let researchers discover this for themselves?

Reply:
Replaced with alternate wording, keeping the last bit.